src/Tribe/Ticket_Object.php

```diff
@@ -684,6 +732,16 @@ public function capacity() {
 				return '';
 			}
 
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: bbda12915c3654ee0630326b3fd5eaaa</sub>